src/_content/schedule/talks/panel-discussion-details-tba.md

```diff
@@ -3,10 +3,30 @@ category: talks
 start_datetime: 2024-09-25 16:30:00-04:00
 difficulty: All
 end_datetime: 2024-09-25 17:15:00-04:00
-permalink: /talks/panel-discussion-details-tba/
+permalink: /talks/panel-discussion-details/
 presenter_slugs:
 - lacey-henschel
 room: Junior Ballroom
-title: Panel Discussion (details TBA)
+title: Panel Discussion
```

Comment:
```suggestion
title: Panel Discussion: Open, friendly, and welcoming: on the history and future of mentoring in Django
```

Reply:
I caught and updated this right after I asked for review!

Reply:
Nice! I'd like to keep the fact that it's a panel in the title somehow, but other than that it's great